src/databricks/labs/remorph/reconcile/query_builder/base.py

```diff
@@ -11,7 +11,7 @@
 
 
 class QueryBuilder(ABC):
-    def __init__(self, table_conf: Table, schema: list[Schema], layer: str, source: str):
+    def __init__(self, table_conf: Table, schema: list[Schema], layer: str, source: Dialects):
```

Comment:
`source` should be DialectType.

https://github.com/databrickslabs/remorph/blob/0496a94f195c7e0cbdb7557f05a3313804e5d015/src/databricks/labs/remorph/config.py

Reply:
done